tornjak-frontend/src/components/cluster-create.tsx

```diff
@@ -224,7 +290,8 @@ class ClusterCreate extends Component {
                   label="Select Cluster Type"
                   titleText="Cluster Type [*required]"
                   onChange={this.onChangeClusterType}
-                  required />
+                //required 
```

Comment:
is this a change we're deciding on or there's another reason for commenting this out?

Reply:
The reason is typescript gives an error `Property 'required' does not exist on type 'IntrinsicAttributes & DropdownProps<string> & { children?: ReactNode; } & RefAttributes<HTMLButtonElement>'.`
But we still have `aria-required="true"` and we have error handling too. Commented it out to explore more.